src/js/sharedEth/connectors.js

```diff
@@ -149,13 +145,15 @@ async function connectShowAccount(eth, showAccount) {
   };
 }
 
-async function connectWalletConnect(eth, disallowAuthDialog = false, desiredNetworkId = 1) {
+async function connectWalletConnect(eth, disallowAuthDialog = false, desiredNetworkId = 1, walletConnectProjectId) {
   const ethProviderName = desiredNetworkId == 3 ? 'ropsten' : 'mainnet';
   const JSONRPC_URL = eth.dataProviders[ethProviderName].host;
   const CHAIN_ID = desiredNetworkId;
 
-  const trxProvider = new WalletConnectProvider({
-    rpc: { [CHAIN_ID]: JSONRPC_URL },
+  const trxProvider = await EthereumProvider.init({
+    projectId: walletConnectProjectId,
+    chains: [1],
+    showQrCode: true,
```

Comment:
Apart from also adding the `@walletconnect/modal` dependency and this code, what else is required to get the QR code modal to show up?